docs/metrics.md

```diff
@@ -0,0 +1,15 @@
+# Exported metrics
+
+## `imageregistry:operations_count:sum`
+
+| Operation | Resource type  |
+| --------- | -------------- |
+| `get`     | blob\|manifest |
+| `create`  | blob\|manifest |
+
+## `imageregistry:imagestreamtags_count:sum`
+
+| Source     | Location         | Description                                                     |
+| ---------- | ---------------- | --------------------------------------------------------------- |
+| `imported` | openshift\|other | Image Stream Tags imported in 'openshift' or 'other' namespaces |
```

Comment:
The quotation marks for `other` may be misleading to think it named `other` namespace.

Reply:
Good point - removed.

Reply:
I would keep quotation in the second column, but remove them from the description.

Reply:
hmm not sure I follow - do you mean the second column should look like:
```
'openshift'|'other'
```
or maybe did you mean we should use backticks, like the first column:
```
`openshift`|`other`
```
?

Reply:
| Source     | Location         | Description                                                     |
| ---------- | ---------------- | --------------------------------------------------------------- |
| `imported` | `openshift`, `other` | Image Stream Tags imported in 'openshift' or other namespaces |

```
| Source     | Location         | Description                                                     |
| ---------- | ---------------- | --------------------------------------------------------------- |
| `imported` | `openshift`, `other` | Image Stream Tags imported in 'openshift' or other namespaces |
```

wdyt?